x-pack/plugins/actions/server/builtin_action_types/slack.ts

```diff
@@ -89,21 +89,21 @@ function renderParameterTemplates(
   };
 }
 
-function valdiateActionTypeConfig(
+function validateActionTypeConfig(
   configurationUtilities: ActionsConfigurationUtilities,
   secretsObject: ActionTypeSecretsType
 ) {
-  let url: URL;
+  const configuredUrl = secretsObject.webhookUrl;
   try {
-    url = new URL(secretsObject.webhookUrl);
+    new URL(configuredUrl);
   } catch (err) {
     return i18n.translate('xpack.actions.builtin.slack.slackConfigurationErrorNoHostname', {
       defaultMessage: 'error configuring slack action: unable to parse host name from webhookUrl',
     });
   }
 
   try {
-    configurationUtilities.ensureHostnameAllowed(url.hostname);
+    configurationUtilities.ensureUriAllowed(configuredUrl);
```

Comment:
I am calling `ensureUriAllowed` both here and in the teams connector to be consistent with how I updated the webhook action. Pagerduty, Jira, Resilient and Servicenow were already all using `ensureUriAllowed`